src/assets/translations/en/main.json

```diff
@@ -1022,7 +1027,12 @@
     "quote": {
       "cantSetSlippage": "We are unable to set a custom slippage (%{userSlippageFormatted}) for %{swapperName}",
       "slippage": "Slippage: %{slippageFormatted}"
-    }
+    },
+    "awaitingPermit2Approval": "Awaiting token transfer",
+    "awaitingSwap": "Awaiting swap via %{swapperName}",
+    "awaitingBridge": "Awaiting bridge via %{bridgeName}",
```

Comment:
Already the case in develop with the current flow but that always confused me - we're losing the source when going to confirm step i.e "Li.Fi - SushiSwap" at quotes time but Li.Fi at preview/confirm time.

<img width="1120" alt="image" src="https://github.com/user-attachments/assets/35209316-83a8-4b51-a822-73ce92708e2d" />
<img width="591" alt="image" src="https://github.com/user-attachments/assets/d645c0fc-577f-42ba-aea1-e66199f7d787" />

Do we want to use the new trade flow as an opportunity to ensure we properly display the source everywhere? i.e `source` instead of `swapperName`/ `bridgeName` 🙏🏽